pageobjects/homepage.js

```diff
@@ -0,0 +1,92 @@
+var page = require('./page');
+var assert = require('assert');
+//var prop = require('./prop');
+
+//Page objects
+var saleButton = '//*[@class="b7afbb84"]'
```

Comment:
Using classes for location an element is not a good idea

Reply:
Locator changed